.github/actions/run-test/action.yml

```diff
@@ -69,7 +78,10 @@ runs:
         client-id: ${{ inputs.flakiness-client-id }}
         tenant-id: ${{ inputs.flakiness-tenant-id }}
         subscription-id: ${{ inputs.flakiness-subscription-id }}
-    - run: ./utils/upload_flakiness_dashboard.sh ./test-results/report.json
+    - run: |
+        echo "::group::./utils/upload_flakiness_dashboard.sh"
```

Comment:
Let's group `Upload blob report` as well?

Reply:
Yeah tested in a private project - doesn't work.